src/components/Carousel/CarouselDots.tsx

```diff
@@ -36,7 +37,7 @@ const centerDotContainer = 11;
 const CarouselDots = forwardRef<HTMLDivElement, CarouselDotsProps>(
   ({ className, maxDots = 9, position = 'inline', numberOfSlides = 0, ...props }, ref) => {
     const { className: baseClassName, ...commonProps } = getCommonProps(props, 'CarouselDots');
-    const componentId = useId();
+    const componentId = uuidv4();
```

Comment:
Will this get recreated every time this component renders? I think this will give us hydration errors. 

Looking at the code below I see we are only using this for a key identifier, which react explicitly says not to do. The way it was defined looked to be static for the whole map. ie. creating one identifier on render and then passing that same id to each map loop. So should be the same of just using index here. 

I think we can just remove and it would be the same.

Reply:
This change does work with React 17 actually but we'll cancel for now since there was an alternate fix for the problem.  The preferred strategy is to get phillips-public to upgrade to React 18